src/app/Command.cpp

```diff
@@ -33,15 +33,14 @@
 namespace chip {
 namespace app {
 
-CHIP_ERROR Command::Init(Messaging::ExchangeManager * apExchangeMgr, InteractionModelDelegate * apDelegate)
+CHIP_ERROR Command::Init(Messaging::ExchangeManager * apExchangeMgr)
```

Comment:
It seems very non-symmetrical to remove Shutdown, but retain Init.

Why can't the body of Init be moved to the Command constructor?

Reply:
Oh, I see.  Shutdown is moved to the inheriting classes.

Would it be cleaner though to still retain Shutdown in the base Command class for it to release the resources it owns?  Inheriting classes can release things specific to their extended implementation and the call the base class Shutdown.

It's just confusing to see the setup and teardown of Command spread between itself and the classes that inherit it.